### ISSUE TITLE
RESTful API implemented app

### ISSUE THREAD

**Issue by kaiPushOrigin** [Glavin001/TMHTAppDev#1]
It would be great if we have a step-by-step tutorial on MEAN Stack by creating a small app. 
- How to transfer data from server to db etc etc.
- What is the importance of Mean Stack..
- REST APi's, HTTP methods..
- What are middlewares

**Comment by Frozenfire92** [comment]
I think rather than focusing on the A(ngular) leave it generic enough to use another framework like Ember or Meteor

**Comment by kaiPushOrigin** [comment]
Totally agree!

**Comment by Glavin001** [comment]
I agree, keep it generic and separated. I want to have a separate few sessions on MVC front-end JavaScript frameworks, which would include Angular, Ember, React, etc.
